app/api/v2/views/user.py

```diff
@@ -0,0 +1,47 @@
+from flask_restplus import Resource, Namespace, fields
+from werkzeug.security import check_password_hash, generate_password_hash
+from flask import request
+from app.api.utils.parcel_validator import UserSchema
+from ..models.user_model import UserModel
+from marshmallow import post_load
+
+v2_user = Namespace('users')
+new_user = v2_user.model('Users', {'email': fields.String('email@example.com'),
+                                   'username': fields.String('test_user'),
+                                   'password': fields.String('test_pass'),
+                                   'confirm_password': fields.String('test_pass')
+
+                                   })
+
+user_login = v2_user.model('Login', {'email': fields.String('email@example.com'),
+                                     'password': fields.String('test_pass')})
+
+
+class User(Resource):
+    @v2_user.expect(new_user)
+    @post_load()
+    def post(self):
+        """route for user registration"""
+        if not request.is_json:
+            return {"msg": "Missing JSON in request"}, 400
+        data = v2_user.payload
+        schema = UserSchema()
+        result = schema.load(data)
+        errors = result.errors
+        error_types = ['username', 'email', 'password']
+        for e in error_types:
```

Comment:
This is the second time I'm repeating about this mistakes. You should make your code readable by leaving a line space before `if` and `for loop` blocks.